### DISCUSSION TITLE
Exam Github Actions

### DISCUSSION THREAD

**Discussion by angelabalaguer17** [v-fidelusaleksander/ghcertified#198] [Study Together!]
Has anyone taken the github actions exam yet? Did the questions help you?

**Comment by tfsjohan** [comment]
Yes I have. I only found GH Certified a couple of hours before my exam, so I did not use it much for studying. But since I answered mostly correctly on the GH Certified questions, it gave me more confidence walking into the exam. 

My advice is to the the practice questions, and for every wrong answer you have, open the documentation (the hint button) and read more about that topic. And repeat. Before taking the exam, I usually spend 10 minutes many times per day doing 10-20 questions. That way it's easy to get done and don't take much time, but you get a lot of practice and repetition.
